src/core/EffectComposer.vue

```diff
@@ -30,7 +30,9 @@ const props = withDefaults(defineProps<EffectComposerProps>(), {
   multisampling: 0,
   stencilBuffer: undefined,
 })
-const { scene, camera, renderer, sizes } = useTresContext()
+const emit = defineEmits(['render'])
```

Comment:
**question:** Is there any advantage in emitting this over using `useLoop` in the parent component? I could imagine, that the frequent emits have some small performance drawbacks. It is also not so nice to have the dev tools view (Component Events) flooded with all the events.

Reply:
That's a good point. We are currently emitting the same on TresCanvas. But we kinda need that for checking the rendering modes

If you check the dev tools you will notice that it doesn't catch the events from `TresCanvas`:

![Screenshot 2024-07-01 at 16 39 19](https://github.com/Tresjs/post-processing/assets/4699008/e19ce3dd-2b21-4160-b693-7a1c39224a37)